packages/fxa-content-server/app/scripts/views/tooltip.js

```diff
@@ -72,7 +72,10 @@ const Tooltip = BaseView.extend({
     this.setPosition();
 
     if (this.dismissible) {
-      this.$el.append('<span class="dismiss" tabindex="2">&#10005;</span>');
+      this.$el.addClass('dismiss-container');
+      this.$el.append(
+        '<button type="button" class="dismiss"+ aria-label="Close"><img src="/images/close-white.svg" alt="&#10005;" aria-hidden="true" /></button>'
```

Comment:
Had some trouble with dynamically changing the colour of the SVG, so created a copy of the SVG with white fill. This can be improved when the component is converted to React.

Reply:
Added a comment about using `fxa-react`'s `close.svg` when this component is converted to React